public/index.html

```diff
@@ -10,7 +10,11 @@
 
         <meta name="description" content="Minecraft team based competitive platform."/>
         <meta name="keywords" content="Minecraft PvP, Minecraft TDM, Minecraft KOTH">
-
+        <meta content="Warzone - Play to Win" property="og:title">
+        <meta content="website" property="og:type">
+        <meta content="https://warz.one/" property="og:url">
+        <meta content="https://avatars1.githubusercontent.com/u/31375932?v=4&s=200" property="og:image">
```

Comment:
Why is the & red?

Reply:
:thinking: It was just getting the Warzone logo based off of the GitHub profile picture.

Maybe just replace it with an image hosted on the site?

Reply:
This shouldn't be a problem.